base/applications/network/ipconfig/ipconfig.c

```diff
@@ -238,6 +239,70 @@ PTCHAR PrintMacAddr(PBYTE Mac)
 }
 
 
+/* convert time_t to localized string */
+_Ret_opt_z_ PTSTR AllocateLocalizedDateTimeFromTimet(_In_ time_t TimeStamp)
+{
+    struct tm* ptm;
+    SYSTEMTIME SystemTime;
+    INT DateCchSize, TimeCchSize, TotalCchSize, i;
+    PTSTR DateTimeString, psz;
+
+    /* Convert Unix time to SYSTEMTIME */
+    /* FIXME: localtime_s may preferred but the declaration not found here */
```

Comment:
Note that it's actually Vista+ and not exported by our CRT...

Reply:
Comment has changed to: localtime_s may preferred if available